package.json

```diff
@@ -37,7 +37,7 @@
     "test": "TS_NODE_TRANSPILE_ONLY=true nyc mocha",
     "test:watch": "TS_NODE_TRANSPILE_ONLY=true mocha --watch",
     "typecheck": "tsc --noEmit",
-    "start": "nodemon"
+    "start": "nodemon & webpack-dev-server --config webpack.dev-server.config --config-name=client"
```

Comment:
Ah, I'd actually forgotten about that when I included webpack-dev-server! Given that (and what I said above about not changing things around too much) it might make sense to remove this (for this PR anyway)?

Reply:
Ah, good point. I'd forgotten we'd done it that way. In that case, I won't bring in a second instance of `webpack-dev-server`.